packages/client/hooks/useMenu.ts

```diff
@@ -53,11 +63,30 @@ const useMenu = <T extends HTMLElement = HTMLButtonElement>(
     return Math.max(40, bbox ? bbox.width : 40)
   }, [originRef.current /* eslint-disable-line react-hooks/exhaustive-deps */])
   const {loadingDelay, loadingDelayRef} = useLoadingDelay()
+
+  const [topOffset, setTopOffset] = useState<number>(0)
+
+  const menuCoords = useMemo(() => {
+    if (
+      portalStatus === PortalStatus.Mounted ||
+      portalStatus === PortalStatus.Entering ||
+      portalStatus === PortalStatus.Entered
+    ) {
+      setTopOffset(cardContainerRef?.current?.scrollTop || 0)
+    }
+    return (coords as {right: number; top: number}).top
+      ? {
+          ...coords,
+          top: (coords as {right: number; top: number}).top - 64 + topOffset
```

Comment:
-1 what's this magic number? 64

Reply:
That was actually a hack we had to use to get the menu in the right position. I will store it in a constant variable for better readability.